packages/fury-adapter-oas3-parser/lib/parser/oas/parseMediaTypeObject.js

```diff
@@ -19,11 +19,9 @@ const unsupportedKeys = ['encoding'];
 const isUnsupportedKey = R.anyPass(R.map(hasKey, unsupportedKeys));
 
 function isJSONMediaType(mediaType) {
-  const type = mediaTyper.parse(mediaType);
-  return (
-    type.type === 'application'
-    && (type.subtype === 'json' || type.suffix === 'json')
-  );
+  const type = contentType.parse(mediaType);
+  const jsonMediaType = /^application\/\S*json$/i;
```

Comment:
I wonder if it would make sense to also use mediaTyper in this case, just wondering if the regex may be fragile in some cases, but I cannot think of any. So up to you really.

```js
const type = mediaTyper.parse(mediaType);
const type = mediaTyper = parse(type.type);
return (
    type.type === 'application'
    && (type.subtype === 'json' || type.suffix === 'json')
)
```